src/lib/cli/templates/rule-script-ts.hbs

```diff
@@ -70,8 +69,8 @@ const rule: IRuleBuilder = {
     },
     meta: {
         docs: {
-            category: '{{category}}',
-            description: '{{description}}'
+            category: `{{category}}`,
```

Comment:
The triple curly brace `{{{variable}}}` avoid to escape the value of the variable.
Using the triple curly brace, maybe you don't need to use the function `escapeSafeString`